### PR TITLE
Reduce unnecessary error logs

### PR DESCRIPTION
**Pull request by yufei-cai**
- MQTT: reduced log level of disconnecting a disconnected client to
  DEBUG.

- AMQP: reduced log level of consumer initialization erros to INFO
  because it can be triggered by incorrect credentials.
  Disabled Qpid JMS Failover log.

- Pub/sub: reduced log level of ddata update failure to WARNING
  before it persists for 3 consecutive clock ticks.

- HTTP: log the entire response on DEBUG level in 1 statement.
  Payload and sensitive headers are hidden automatically.

### CODE REVIEW

File: connectivity/service/src/main/java/org/eclipse/ditto/connectivity/service/messaging/amqp/AmqpClientActor.java [hunk 1]
```diff
@@ -397,32 +396,6 @@ protected void cleanupFurtherResourcesOnConnectionTimeout(final BaseClientState
...
-                return Objects.equals(sender, connectConnectionHandler);
-            case DISCONNECTING:
-                return Objects.equals(sender, disconnectConnectionHandler);
-            case TESTING:
-            default:
-                // no need to check testConnectionHandler because test runs only once during this actor's lifetime
-                // ignore random events by default - they could come from a connection handler that is already dead
-                return false;
-        }
-    }
```
Comment (thjaeckle) [inline connectivity/service/src/main/java/org/eclipse/ditto/connectivity/service/messaging/amqp/AmqpClientActor.java:424]: @yufei-cai why did you remove this? 
If it's not required, it's sure a good thing, but I don't understand how you came to the conclusion to remove it :)

Comment (yufei-cai) [reply connectivity/service/src/main/java/org/eclipse/ditto/connectivity/service/messaging/amqp/AmqpClientActor.java:424]: The reason is that connection failures can now come from consumer actors in connected state. Instead of maintaining the "isEventUpToDate"  table for this change and all future changes I think it better to risk restarting due to stale events than to ignore relevant events.
